BaboViolent2/Code/FileIO.h

```diff
@@ -60,10 +60,10 @@ class FileIO
 	virtual unsigned char * getUByteArray(int size);
 	virtual int getInt(); // short
 	virtual unsigned int getUInt(); // unsigned short
-	virtual long getLong();
-	virtual long * getLongArray(int size);
-	virtual unsigned long getULong();
-	virtual unsigned long * getULongArray(int size);
```

Comment:
Were long meant to be 64 bits back then? Shouldn't we replace with int64_t?
I guess you tested...